Sources/BBCode/Spoiler1.php

```diff
@@ -67,7 +67,9 @@ class Spoiler1 extends BBCode
 	public ?string $before = '<span class="bbc_inline_spoiler" title="{txt_spoiler_toggle}"><button type="button" title="{txt_spoiler_toggle}" tabindex="0"></button><span>';
 
 	/**
+	 * @var string|null
 	 *
+	 * HTML to insert after the BBCode's content
```

Comment:
```suggestion
	 *
```

This change is unnecessary because the entire contents of the DocBlock are inherited.